Tools/CMake/AMReX_Options.cmake

```diff
@@ -90,15 +90,33 @@ endif ()
 print_option( DIM )
 
 #
-# Programming models  ========================================================
+# Parallel backends    ========================================================
```

Comment:
```suggestion
# Parallel, on-node compute backends    =======================================
```

Reply:
I would rather keep it more general: that comment marks the beginning of the section devoted to every parallel backend, including MPI

Reply:
Okay, I would separate on-node computing backends and intra-node comm, but this is no big deal.